### ISSUE TITLE
Convert complicated npm scripts to bash scripts

### ISSUE THREAD

**Issue by danoc** [optimizely/oui#435]
Some of the npm scripts are hard to follow due to the formatting restrictions that JSON imposes.

We already have a `scripts/` directory, so we can convert some of them to bash scripts that live in there.

**Comment by daverau-optimizely** [comment]
Hey @danoc, is the main benefit to avoid extra escaping necessary in a gulp script vs bash? If so, sounds good.

**Comment by danoc** [comment]
@daverau-optimizely – It's mostly because our scripts are fairly complicated and JSON doesn't allow line breaks or comments.

**Comment by danoc** [comment]
Looked into this and it's not worth doing because it adds tons of complexity. I refactored a bunch of the complicated NPM scripts into less-complicated NPM scripts in #631. I think that's fine for now.
